flask_potion/contrib/alchemy/manager.py

```diff
@@ -256,88 +256,87 @@ def _query_get_first(self, query):
         except NoResultFound:
             raise IndexError()
 
-    def create(self, properties, commit=True):
+    def create(self, properties):
         # noinspection properties
         item = self.model(**properties)
 
         before_create.send(self.resource, item=item)
-
         session = self._get_session()
-
-        try:
-            session.add(item)
-            self.commit_or_flush(commit)
-        except IntegrityError as e:
-            session.rollback()
-
-            if hasattr(e.orig, 'pgcode'):
-                if e.orig.pgcode == "23505":  # duplicate key
-                    raise DuplicateKey(detail=e.orig.diag.message_detail)
-
-            if current_app.debug:
-                raise BackendConflict(
-                    debug_info=dict(
-                        exception_message=str(e), statement=e.statement, params=e.params
+        with session.begin_nested():
```

Comment:
WIP

To summarize my understanding.

We want to avoid having `Potion` (that uses `flask-sqlalchemy`) committing the current transaction when models are created/updated/removed. The previous code is committing because of the default parameter `commit`, e.g `create(self, properties, commit=True)`.

At this point